inspirehep/modules/search/api.py

```diff
@@ -90,7 +90,10 @@ def mget(self, uuids, **kwargs):
                 body={'ids': uuids},
                 **kwargs
             )
-            results = [document['_source'] for document in documents['docs']]
+            results = [
+                document['_source'] for document in documents['docs']
+                if document.get('found')
```

Comment:
So when there are no real matches, there is still one document but with no `found` key?

Reply:
kind of:
```
{
    u'_id': u'7f6bd778-c2f8-4ddf-a98d-34db4024544c',
    u'_index': u'records-hep',
    u'_type': u'hep',
    u'found': False
}
```

Reply:
Ah I see 👍